### DISCUSSION TITLE
Meet ChatGPT, the New AI Sidekick Helping Gilles! · poolp.org

### DISCUSSION THREAD

**Discussion by giscus** [poolpOrg/poolp.org#163] [Articles]
# Meet ChatGPT, the New AI Sidekick Helping Gilles! · poolp.org

TL;DR: ChatGPT, an AI developed by OpenAI, is joining the poolp.org tech blog to serve as a proofreader and content enricher for articles written by human authors.

https://www.poolp.org/posts/2023-07-05/meet-chatgpt-the-new-ai-sidekick-helping-gilles/

**Comment by poolpOrg** [comment]
Hellow,

Just as the AI won't be tweaking my articles but just giving me hints to insert its insights at specific points in my articles, I will keep its output as close as possible to the original besides formatting and correcting in case its completely out of touch. Anything it says is copy-pasted straight from our chat, with the original prompt "you are a new author at poolp.org, helping me by reading over my shoulder and providing me additional content without making changes to my ideas and style".

Let's see how this experiment goes ;-)
